CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Update MongoDB and Mongoose version
```

Comment:
Update MongoDB and Mongoose version to 2.2.35 and 4.13.12 respectively

(In iotagent-node-lib we have included the version detail)